guaranteed-message-delivery/consumer.bal

```diff
@@ -0,0 +1,71 @@
+import ballerina/http;
+import ballerina/log;
+import ballerinax/rabbitmq;
+import ballerinax/twilio;
+
+type MessageContent record {|
+    *ReservationRequest;
+    string category;
+|};
+
+type RabbitMqMessage record {|
+    *rabbitmq:AnydataMessage;
+    MessageContent content;
+|};
+
+configurable string fromNumber = ?;
+configurable string accountSId = ?;
+configurable string authToken = ?;
+
+final http:Client hospitalBackend = check initializeHttpClient();
+
+final twilio:Client twilioEp = check initializeTwilioClient();
+
+function initializeHttpClient() returns http:Client|error => new ("http://localhost:9090");
+
+function initializeTwilioClient() returns twilio:Client|error => new ({
+    twilioAuth: {
+        accountSId,
+        authToken
+    }
+});
+
+@rabbitmq:ServiceConfig {
+    queueName
+}
+service rabbitmq:Service on new rabbitmq:Listener(rabbitmq:DEFAULT_HOST, rabbitmq:DEFAULT_PORT) {
```

Comment:
Isn't `rabbitmq:Service` optional here?

Reply:
But in the docs, we have it. Won't it be a little bit confusing if someone refers the doc with the tutorial?

Reply:
IMO, the docs don't require it either. This is similar to omitting the service type in http services too.